examples/support/__init__.py

```diff
@@ -173,6 +174,8 @@ def write_experiment_json(
     postprocess_func : Optional[Callable[[dict], dict]]
         If provided, this is called with the experiment document for any postprocessing.  The callable should return
         what is to be written as the experiment document.
+    default_shape : Tuple[int, int]
```

Comment:
docstring should list the default value for an optional parameter (see [sections point 4](https://numpydoc.readthedocs.io/en/latest/format.html#sections))

Reply:
Fixed.